### ISSUE TITLE
Doesn't work with neovim-qt

### ISSUE THREAD

**Issue by chrisburel** [finestructure/Hummingbird#28]
I have a brand new install of macOS Catalina on a MacBook Pro. I installed hummingbird with homebrew, then installed neovim-qt with:

```
brew tap equalsraf/homebrew-neovim-qt
brew install neovim-qt
```

I then ran nvim-qt from the command line, and hummingbird is unable to move or resize the window. It is as though the neovim window is not there, if I have a window stack of Terminal underneath the Neovim window, hummingbird will move the Terminal window that is behind the Neovim window. I would expect that Neovim is a normal Qt application, I wouldn't expect that it is doing anything strange that would prevent Hummingbird from seeing its window.

**Comment by finestructure** [comment]
Hi @chrisburel !

Thanks for reporting this. I'm pretty sure this is a duplicate, or at least would be fixed in the same way as #5 .

There are certain windows, apparently in particular when implemented via cross-platform frameworks, that are "invisible" to the accessibility mechanisms HB uses to find and manipulate them.

Hopefully this can be addressed by what's laid out in #5 at some point!

**Comment by finestructure** [comment]
Housekeeping: Closing this as a duplicate of #5
